### PR TITLE
Added Enhanced Video Settings minus terrain stuff

### PR DESCRIPTION
**Pull request by GordonWeedman**
To alleviate the fears of people removing grass from the terrain, I removed all custom terrain detail settings.
This has been done with the permission of the original author.

### CODE REVIEW

**Comment by Cuel** [comment]
Nice, I'll take a look later

File: addons/common/config.cpp [hunk 1]
```diff
@@ -5,7 +5,12 @@ class CfgPatches {
     units[] = {};
     weapons[] = {};
     requiredVersion = 0.1;
-    requiredAddons[] = {"fpa_main"};
+    requiredAddons[] = 
+	{
+		"fpa_main",
+		"A3_Data_F",
```
Comment (Cuel) [inline addons/common/config.cpp]: These two are not needed, fpa_main already requires full a3 + ace

File: addons/common/config.cpp [hunk 2]
```diff
@@ -5,7 +5,12 @@ class CfgPatches {
     units[] = {};
     weapons[] = {};
     requiredVersion = 0.1;
-    requiredAddons[] = {"fpa_main"};
+    requiredAddons[] = 
+	{
+		"fpa_main",
+		"A3_Data_F",
+		"A3_UI_f"
+	};
```
Comment (Cuel) [inline addons/common/config.cpp]: These two above are not needed, fpa_main already requires full a3 + ace

File: addons/common/config.cpp [hunk 3]
```diff
@@ -42,3 +47,183 @@ class RscDisplayCurator {
         };
     };
 };
+class CfgVideoOptions
```
Comment (Cuel) [inline addons/common/config.cpp]: Can you move this into CfgVideoOptions and #include it instead?

Comment (GordonWeedman) [reply addons/common/config.cpp]: Done

File: addons/common/config.cpp (hunk 2 above)
Comment (GordonWeedman) [reply addons/common/config.cpp]: Done

**Pull request merged**